app/src/lib/editors/win32.ts

```diff
@@ -479,6 +479,15 @@ const editors: WindowsExternalEditor[] = [
     displayNamePrefix: 'DataSpell ',
     publishers: ['JetBrains s.r.o.'],
   },
+  {
+    name: 'Pulsar',
+    registryKeys: [
+      CurrentUserUninstallKey('0949b555-c22c-56b7-873a-a960bdefa81f')
```

Comment:
Let's please the linter!
```suggestion
      CurrentUserUninstallKey('0949b555-c22c-56b7-873a-a960bdefa81f'),
```